draft-bbs-signatures.md

```diff
@@ -554,10 +557,10 @@ Procedure:
 
 ## SpkGen
 
-A signature proof of knowledge generating algorithm that creates a zero-knowledge proof of knowledge of a signature while selectively disclosing messages from a signature given a vector of messages, a vector of indices of the revealed messages, the signer's public key, and a nonce.
+A signature proof of knowledge generating algorithm that creates a zero-knowledge proof of knowledge of a signature while selectively disclosing messages from a signature given a vector of messages, a vector of indices of the revealed messages, the signer's public key, and a presentation message.
 
 ```
-spk = SpkGen(PK, (msg[1],...,msg[L]), RIdxs, signature, nonce)
+spk = SpkGen(PK, (msg[1],...,msg[L]), RIdxs, signature, pm)
```

Comment:
Wouldn't we also need a nonce here to avoid replay? Or is the intention to use PM as the nonce when necessary?

Reply:
The PM is in reality an arbitrary data structure that includes multiple pieces information such as a nonce, the guidance for this I think is best addressed through a revision to @christianpaquin PR #21.